src/xenia/kernel/xam/user_profile.cc

```diff
@@ -14,7 +14,9 @@ namespace kernel {
 namespace xam {
 
 UserProfile::UserProfile() {
-  xuid_ = 0xBABEBABEBABEBABE;
+  // Write XUID with 0xE as it indicates the profile is offline.
+  // https://github.com/hetelek/Velocity/blob/master/XboxInternals/Account/Account.cpp#L77
+  xuid_ = 0xE000000004CE0DB0;
```

Comment:
I can understand the upper `0xE`, but what about the `0x4CE0DB0`?

Reply:
The 0x4CE0DB0 was just ripped from an actual XUID off a console as filler data, it's not relevant I just didn't want to write complete 0'd out data as I wasn't sure if a game would be checking for this or not.

Reply:
Is it some special kind of a user ID consistent across different consoles, or just your personal ID? This should be explained in comments preferably.

Reply:
It's not specific or special in any way outside of the upper 0xE a real XUID was simply used incase any game actually had decided to check against it, or instead of leaving those bytes as 0s.

Reply:
The 0xE0 is checked against, we used the XUID 0xE000BABEBABEBABE in canary with no adverse effects